src/ErrorsServiceProvider.php

```diff
@@ -21,7 +21,7 @@ class ErrorsServiceProvider extends ServiceProvider
     public function register()
     {
         $this->publishes([
-            __DIR__.'/publish/errors' => base_path('views/errors'),
+            __DIR__.'/publish/errors' => base_path('resources/views/errors'),
```

Comment:
This is more of a choice kind of comment but both base_path could be replaced by resource_path and public_path respectively.